pkg/downloader/manager.go

```diff
@@ -246,7 +246,7 @@ func (m *Manager) downloadAll(deps []*chart.Dependency) error {
 	}
 
 	destPath := filepath.Join(m.ChartPath, "charts")
-	tmpPath := filepath.Join(m.ChartPath, "tmpcharts")
+	tmpPath := filepath.Join(m.ChartPath, fmt.Sprintf("tmpcharts-%d", os.Getpid()))
```

Comment:
While this generally works, there are some tests to make sure we don't leave the _tmpcharts_ directory behind. The tests need to be updated to ensure the directory under the new name isn't left behind. You can find these by searching for _tmpcharts_ in the codebase. One example is in the test `TestDependencyUpdateCmd_DoNotDeleteOldChartsOnError`.

Can you please make sure the tests are updated for this change.

Reply:
Thanks for pointing that out, fixed it